### PR TITLE
Ava GUI: Restructure `Ryujinx.Ava`

### PR DESCRIPTION
**Pull request by IsaacMarovitz**
**Changes:**
- `Ryujinx.Ava.Ui` renamed to `Ryujinx.Ava.UI` to match Avalonia namespace styling
- `Ryujinx.Ava` project reorganised
- Fixes stylistic inconsistencies between AXAML files

Superceeds part of #3997

Should have no effect on functionality.

### CODE REVIEW

File: Ryujinx.Ava/UI/ViewModels/SettingsViewModel.cs [hunk 1]
```diff
@@ -199,7 +200,7 @@ public float Volume
 
         public DateTimeOffset DateOffset { get; set; }
         public TimeSpan TimeOffset { get; set; }
-        public AvaloniaList<TimeZone> TimeZones { get; set; }
+        public AvaloniaList<Models.TimeZone> TimeZones { get; set; }
```
Comment (TSRBerry) [inline Ryujinx.Ava/UI/ViewModels/SettingsViewModel.cs:203]:
```suggestion
        public AvaloniaList<TimeZone> TimeZones { get; set; }
```

File: Ryujinx.Ava/UI/ViewModels/SettingsViewModel.cs [hunk 2]
```diff
@@ -231,7 +232,7 @@ public SettingsViewModel(VirtualFileSystem virtualFileSystem, ContentManager con
         public SettingsViewModel()
         {
             GameDirectories = new AvaloniaList<string>();
-            TimeZones = new AvaloniaList<TimeZone>();
+            TimeZones = new AvaloniaList<Models.TimeZone>();
```
Comment (TSRBerry) [inline Ryujinx.Ava/UI/ViewModels/SettingsViewModel.cs:235]:
```suggestion
            TimeZones = new AvaloniaList<TimeZone>();
```

File: Ryujinx.Ava/UI/ViewModels/SettingsViewModel.cs [hunk 3]
```diff
@@ -288,7 +289,7 @@ public void LoadTimeZones()
 
                 string abbr2 = abbr.StartsWith('+') || abbr.StartsWith('-') ? string.Empty : abbr;
 
-                TimeZones.Add(new TimeZone($"UTC{hours:+0#;-0#;+00}:{minutes:D2}", location, abbr2));
+                TimeZones.Add(new Models.TimeZone($"UTC{hours:+0#;-0#;+00}:{minutes:D2}", location, abbr2));
```
Comment (TSRBerry) [inline Ryujinx.Ava/UI/ViewModels/SettingsViewModel.cs:292]:
```suggestion
                TimeZones.Add(new TimeZone($"UTC{hours:+0#;-0#;+00}:{minutes:D2}", location, abbr2));
```

File: Ryujinx.Ava/UI/ViewModels/UserProfileViewModel.cs [hunk 4]
```diff
@@ -5,27 +5,28 @@
...
+namespace Ryujinx.Ava.UI.ViewModels
 {
     public class UserProfileViewModel : BaseModel, IDisposable
     {
         private readonly NavigationDialogHost _owner;
 
-        private UserProfile _selectedProfile;
-        private UserProfile _highlightedProfile;
+        private Models.UserProfile _selectedProfile;
+        private Models.UserProfile _highlightedProfile;
```
Comment (TSRBerry) [inline Ryujinx.Ava/UI/ViewModels/UserProfileViewModel.cs:24]:
```suggestion
        private UserProfile _selectedProfile;
        private UserProfile _highlightedProfile;
```

File: Ryujinx.Ava/UI/ViewModels/UserProfileViewModel.cs [hunk 5]
```diff
@@ -5,27 +5,28 @@
...
 
         public UserProfileViewModel()
         {
-            Profiles = new ObservableCollection<UserProfile>();
-            LostProfiles = new ObservableCollection<UserProfile>();
+            Profiles = new ObservableCollection<Models.UserProfile>();
+            LostProfiles = new ObservableCollection<Models.UserProfile>();
```
Comment (TSRBerry) [inline Ryujinx.Ava/UI/ViewModels/UserProfileViewModel.cs:29]:
```suggestion
            Profiles = new ObservableCollection<UserProfile>();
            LostProfiles = new ObservableCollection<UserProfile>();
```

File: Ryujinx.Ava/UI/ViewModels/UserProfileViewModel.cs [hunk 6]
```diff
@@ -35,11 +36,11 @@ public UserProfileViewModel(NavigationDialogHost owner) : this()
             LoadProfiles();
         }
 
-        public ObservableCollection<UserProfile> Profiles { get; set; }
+        public ObservableCollection<Models.UserProfile> Profiles { get; set; }
```
Comment (TSRBerry) [inline Ryujinx.Ava/UI/ViewModels/UserProfileViewModel.cs:39]:
```suggestion
        public ObservableCollection<UserProfile> Profiles { get; set; }
```

File: Ryujinx.Ava/UI/ViewModels/UserProfileViewModel.cs [hunk 7]
```diff
@@ -35,11 +36,11 @@ public UserProfileViewModel(NavigationDialogHost owner) : this()
...
 
-        public ObservableCollection<UserProfile> LostProfiles { get; set; }
+        public ObservableCollection<Models.UserProfile> LostProfiles { get; set; }
```
Comment (TSRBerry) [inline Ryujinx.Ava/UI/ViewModels/UserProfileViewModel.cs:41]:
```suggestion
        public ObservableCollection<UserProfile> LostProfiles { get; set; }
```

File: Ryujinx.Ava/UI/ViewModels/UserProfileViewModel.cs [hunk 8]
```diff
@@ -35,11 +36,11 @@ public UserProfileViewModel(NavigationDialogHost owner) : this()
...
 
-        public UserProfile SelectedProfile
+        public Models.UserProfile SelectedProfile
```
Comment (TSRBerry) [inline Ryujinx.Ava/UI/ViewModels/UserProfileViewModel.cs:43]:
```suggestion
        public UserProfile SelectedProfile
```

File: Ryujinx.Ava/UI/ViewModels/UserProfileViewModel.cs [hunk 9]
```diff
@@ -56,7 +57,7 @@ public UserProfile SelectedProfile
 
         public bool IsHighlightedProfileDeletable => _highlightedProfile != null && _highlightedProfile.UserId != AccountManager.DefaultUserId;
 
-        public UserProfile HighlightedProfile
+        public Models.UserProfile HighlightedProfile
```
Comment (TSRBerry) [inline Ryujinx.Ava/UI/ViewModels/UserProfileViewModel.cs:60]:
```suggestion
        public UserProfile HighlightedProfile
```

File: Ryujinx.Ava/UI/ViewModels/UserProfileViewModel.cs [hunk 10]
```diff
@@ -80,7 +81,7 @@ public void LoadProfiles()
 
             foreach (var profile in profiles)
             {
-                Profiles.Add(new UserProfile(profile, _owner));
+                Profiles.Add(new Models.UserProfile(profile, _owner));
```
Comment (TSRBerry) [inline Ryujinx.Ava/UI/ViewModels/UserProfileViewModel.cs:84]:
```suggestion
                Profiles.Add(new UserProfile(profile, _owner));
```

File: Ryujinx.Ava/UI/ViewModels/UserProfileViewModel.cs [hunk 11]
```diff
@@ -128,20 +129,20 @@ public void LoadProfiles()
 
             foreach(var account in lostAccounts)
             {
-                LostProfiles.Add(new UserProfile(new HLE.HOS.Services.Account.Acc.UserProfile(account, "", null), _owner));
+                LostProfiles.Add(new Models.UserProfile(new HLE.HOS.Services.Account.Acc.UserProfile(account, "", null), _owner));
```
Comment (TSRBerry) [inline Ryujinx.Ava/UI/ViewModels/UserProfileViewModel.cs:132]:
```suggestion
                LostProfiles.Add(new UserProfile(new HLE.HOS.Services.Account.Acc.UserProfile(account, "", null), _owner));
```

File: Ryujinx.Ava/UI/ViewModels/UserProfileViewModel.cs [hunk 12]
```diff
@@ -128,20 +129,20 @@ public void LoadProfiles()
...
             }
         }
 
         public void AddUser()
         {
-            UserProfile userProfile = null;
+            Models.UserProfile userProfile = null;
```
Comment (TSRBerry) [inline Ryujinx.Ava/UI/ViewModels/UserProfileViewModel.cs:138]:
```suggestion
            UserProfile userProfile = null;
```

File: Ryujinx.Ava/UI/ViewModels/UserProfileViewModel.cs [hunk 13]
```diff
@@ -128,20 +129,20 @@ public void LoadProfiles()
...
 
             _owner.Navigate(typeof(UserEditor), (this._owner, userProfile, true));
         }
 
         public async void ManageSaves()
         {
-            UserProfile userProfile = _highlightedProfile ?? SelectedProfile;
+            Models.UserProfile userProfile = _highlightedProfile ?? SelectedProfile;
```
Comment (TSRBerry) [inline Ryujinx.Ava/UI/ViewModels/UserProfileViewModel.cs:145]:
```suggestion
            UserProfile userProfile = _highlightedProfile ?? SelectedProfile;
```

Comment (TSRBerry) [inline Ryujinx.Ava/UI/ViewModels/UserProfileViewModel.cs:145]:
```suggestion
            UserProfile userProfile = _highlightedProfile ?? SelectedProfile;
```

File: Ryujinx.Ava/UI/Windows/SettingsWindow.axaml.cs [hunk 14]
```diff
@@ -192,7 +193,7 @@ private void TimeZoneBox_OnTextChanged(object sender, EventArgs e)
         {
             if (sender is AutoCompleteBox box)
             {
-                if (box.SelectedItem != null && box.SelectedItem is TimeZone timeZone)
+                if (box.SelectedItem != null && box.SelectedItem is Models.TimeZone timeZone)
```
Comment (TSRBerry) [inline Ryujinx.Ava/UI/Windows/SettingsWindow.axaml.cs:196]:
```suggestion
                if (box.SelectedItem != null && box.SelectedItem is TimeZone timeZone)
```

Comment (TSRBerry) [inline Ryujinx.Ava/UI/Windows/SettingsWindow.axaml.cs:196]:
```suggestion
                if (box.SelectedItem != null && box.SelectedItem is TimeZone timeZone)
```

**Review by TSRBerry — CHANGES_REQUESTED** [review]

File: Ryujinx.Ava/UI/Helpers/EmbeddedWindow.cs [hunk 15]
```diff
@@ -70,11 +70,13 @@ protected override IPlatformHandle CreateNativeControlCore(IPlatformHandle paren
             {
                 return CreateLinux(parent);
             }
-            else if (OperatingSystem.IsWindows())
+
```
Comment (TSRBerry) [inline Ryujinx.Ava/UI/Helpers/EmbeddedWindow.cs:73]: I know it's not necessary, but I'd prefer to keep these as `else if`s to be explicit here.

File: Ryujinx.Ava/UI/ViewModels/SettingsViewModel.cs [hunk 16]
```diff
@@ -41,7 +42,6 @@ internal class SettingsViewModel : BaseModel
         private float _customResolutionScale;
         private int _resolutionScale;
         private int _graphicsBackendMultithreadingIndex;
-        private float _previousVolumeLevel;
```
Comment (TSRBerry) [inline Ryujinx.Ava/UI/ViewModels/SettingsViewModel.cs:44]: Wasn't this used in `SettingsWindow.axaml`?

Comment (IsaacMarovitz) [reply Ryujinx.Ava/UI/ViewModels/SettingsViewModel.cs:44]: It's used in GTK UI, it is not used in Ava

**Review by TSRBerry — APPROVED** [review]
Lgtm, nicely done!

**Review by emmauss — APPROVED** [review]

**Review by AcK77 — APPROVED** [review]

**Pull request merged**